conda-recipe/meta.yaml

```diff
@@ -1,20 +1,23 @@
-{% set data = load_setup_py_data() %}
+{% set package_name = "pcdsdevices" %}
+{% set import_name = "pcdsdevices" %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
 
 package:
-  name: pcdsdevices
-  version : {{ data.get('version') }}
+  name: {{ package_name }}
+  version : {{ version }}
 
```

Comment:
And here's just a little bit more detail about py311 build failure, which should be out of scope for this PR:
https://github.com/pcdshub/pcdsdevices/actions/runs/3907251443/jobs/6676280916#step:11:846
```
Mamba failed to solve:
 - pcdsdevices 7.1.1.dev84+g5b42573cbd py_0
 - bluesky >=1.6.4
 - happi
 - jsonschema
 - lightpath >=1.0.0
 - matplotlib-base
 - numpy
 - ophyd >=1.5.1
 - pcdscalc >=0.2.0
 - pcdsutils >=0.4.0
 - pint
 - prettytable
 - pyepics >=3.4.2
 - python >=3.9
 - pytmc >=2.7.0
 - pyyaml
 - schema
 - scipy
 - python
 - python 3.11.*

with channels:
 - file:///home/runner/conda-bld

The reported errors are:
- Encountered problems while solving:
-   - package matplotlib-base-3.5.2-py37he341ac4_0 requires python_abi 3.7 *_pypy37_pp73, but none of the providers can be installed
```

The last error message is nonsensical, the only odd thing I see is `python` being specified twice

Reply:
The weird thing there is that matplotlib-base is on py37 (see the build string) for some reason instead of py311...

Matplotlib-base does have 3.11 builds, it seems:
https://anaconda.org/conda-forge/matplotlib-base/files